docs/config/build-options.md

```diff
@@ -128,6 +128,10 @@ It should only be used when you are targeting a non-mainstream browser.
 One example is Android WeChat WebView, which supports most modern JavaScript features but not the [`#RGBA` hexadecimal color notation in CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#rgb_colors).
 In this case, you need to set `build.cssTarget` to `chrome61` to prevent vite from transform `rgba()` colors into `#RGBA` hexadecimal notations.
 
+::: info
+Please note that currently `build.cssTarget` is intended to be recognized only by Esbuild. When the CSS minifier in your project is set to 'lightningcss' in `build.cssMinify`, then `build.cssTarget` is no longer effective and you must write targets in `css.lightningcss.targets`. [Reference](https://github.com/vitejs/vite/issues/17900).
+:::
+
```

Comment:
Hey, thanks for the PR.
I think I've mis-spoken a little in https://github.com/vitejs/vite/issues/17900#issuecomment-2297856001 and `cssTarget` indeed influences lightningcss targets for build-time minification and it override `css.lightningcss.targets`:

https://github.com/vitejs/vite/blob/b1ecdaf6594b48d2fcbff8682e9ef68916806089/packages/vite/src/node/plugins/css.ts#L1809-L1817

Note that the reason why https://github.com/vitejs/vite/issues/17900#issuecomment-229785600 got an error is that Vite allows using lightningcss in two distinct step `css.transformer: "lightningcss"` and `build.cssMinify: "lightningcss"`. `css.transformer` is used to transform each css file (both during dev and build) while `build.cssMinify` is used to minify combined css chunk/bundle at the final step (only during build). Then the error was due to first step's configuration `css.lightningcss.targets`.

I'm not sure how to word this properly in the documentation, but maybe something like:

> When using both `css.transformer: "lightningcss"` and `build.cssMinify: "lightningcss"`, you can use `css.lightningcss.targets` to ...

Or maybe we can scratch line 1812 so that this PR's documentation update becomes correct.